package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "event-manager",
+  "version": "1.0.0",
+  "description": "This App allows client post events online and also grants admin access level to the site administrators to smoothly run the the site.",
+  "main": "server/app.js",
+  "scripts": {
+    "start": "babel-watch server/app.js",
+    "babel-node": "babel-node --",
```

Comment:
What would you be using this script for?

Reply:
babel-watch works just like nodemon, it watches for changes in my code after I save it and restarts the server

Reply:
I am talking of the `babel-node` script

Reply:
ohh, I see it, some real mistake there. Would take that off